### PR TITLE
Some tiny improvements

### PR DESCRIPTION
**Pull request by data-man**
Thanks for the cool tool!

I made a few tiny improvements:

- Added CVS and Bazaar to the excluded dirs
- Added D, Nim, HTML, CSS, Go, Rust, Markdown, reStructuredText, XML, JSON, CSV, Delphi, Pascal, Kotlin, TypeScript, txt, Julia, Zig to the default cfg
- Sort the list of languages ​​in alphabetical order
- Changed `-msse2` to `-mtune=native` in the makefile

### CODE REVIEW

File: src/init.cpp [hunk 1]
```diff
@@ -51,6 +60,8 @@ void initProject(Output* output, const char* name, const char* file, const char*
 	{
 		out << "# " << kDefaultLanguages[i] << std::endl;
 		out << "include \\.(" << kDefaultLanguages[i + 1] << ")$" << std::endl;
+		out << "# exclude something/$" << std::endl;
```
Comment (zeux) [inline src/init.cpp]: Do we need this, and the line below?

Comment (zeux) [reply src/init.cpp]: Right, but why would you need an exclude after each include for each language? Also - if the intent is to provide a template for folder exclusion - it should be exclude ^something/

Comment (data-man) [reply src/init.cpp]: > why would you need an exclude after each include for each language?

My use case:
- gen cfg
- edit cfg - remove unneeded languages
- excluded folder left in place

> it should be exclude ^something/

Thanks, corrected!

Comment (zeux) [reply src/init.cpp]: I still don't understand why you need an exclude line *per language*. Can you post the .cfg file generated after your change?

Comment (data-man) [reply src/init.cpp]: Sure.
```
path /home/Repos/C

# C/C++
include \.(cpp|cxx|cc|c|hpp|hxx|hh|h|inl)$
# exclude ^some folder/

# C#, VB.NET
include \.(cs|vb)$
# exclude ^some folder/

# D
include \.(d)$
# exclude ^some folder/

# Delphi, Pascal
include \.(dpr|pas|pp|inc|dfm|lfm|lpi|lpr|dpk|dproj|spp)$
# exclude ^some folder/

# F#, OCaml, Haskell
include \.(fs|fsi|fsx|ml|mli|hs)$
# exclude ^some folder/

# Go
include \.(go)$
# exclude ^some folder/

# HTML, CSS
include \.(htm|html|css|sass|scss)$
# exclude ^some folder/

# Java, JavaScript, Kotlin, TypeScript
include \.(java|js|kt|kts|ts|tsx)$
# exclude ^some folder/

# Julia
include \.(jl)$
# exclude ^some folder/

# Lua, Squirrel
include \.(lua|nut)$
# exclude ^some folder/

# Markdown, reStructuredText, simple text
include \.(md|rst|txt)$
# exclude ^some folder/

# Nim
include \.(nim)$
# exclude ^some folder/

# Objective C/C++
include \.(m|mm)$
# exclude ^some folder/

# Perl, Python, Ruby
include \.(pl|py|pm|rb)$
# exclude ^some folder/

# PHP, ActionScript
include \.(php|as)$
# exclude ^some folder/

# Rust
include \.(rs)$
# exclude ^some folder/

# Shaders
include \.(hlsl|glsl|cg|fx|cgfx)$
# exclude ^some folder/

# XML, JSON, CSV
include \.(xml|json|csv)$
# exclude ^some folder/

# Zig
include \.(zig)$
# exclude ^some folder/
```

**Pull request merged**